src/app/app.module.ts

```diff
@@ -16,6 +16,8 @@ import { VerificationPage } from '../pages/verification/verification';
 import { SearchPipe } from '../pipes/search';
 import { PictureService } from '../services/picture';
 import { MyApp } from './app.component';
+import { AgmCoreModule } from 'angular2-google-maps/core';
+import { NewLocationMessageComponent } from '../pages/location-message/location-message';
```

Comment:
This is a component which is part of the messages page, **not** a page by itself, therefore it should be moved to `pages/messages/location-message`